all-together.ipynb

```diff
@@ -0,0 +1,775 @@
+{
```

Comment:
<p><code>sn</code>  and <code>snp</code>  are very hard to understand. Consider renaming them to <code>snippet</code>  and <code>snippets</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/JetBrains-Research/metrics-evaluation/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>